src/Sarif.Multitool/MergeCommand.cs

```diff
@@ -24,6 +24,14 @@ public  int Run(MergeOptions mergeOptions)
         {
             try
             {
+                string outputDirectory = mergeOptions.OutputFolderPath ?? Environment.CurrentDirectory;
+                string outputFilePath = Path.Combine(outputDirectory, GetOutputFileName(mergeOptions));
+
+                if (!DriverUtilities.ReportWhetherOutputFileCanBeCreated(outputFilePath, mergeOptions.Force, _fileSystem))
+                {
+                    return -1;
```

Comment:
>-1 [](start = 27, length = 2)

It would be nice if we had a set of common return codes for Run().

Reply:
Sorry that should have been `return 1;` Fixed.

---
In reply to: [312280114](https://github.com/microsoft/sarif-sdk/pull/1625#discussion_r312280114) [](ancestors = 312280114)